cli/src/main/java/com/liferay/blade/cli/command/UpdateCommand.java

```diff
@@ -322,7 +322,7 @@ public void execute() {
 
 					Runtime runtime = Runtime.getRuntime();
 
-					runtime.exec("cmd /c start \"" + batPath + "\"");
+					runtime.exec("cmd /c start \"\" " + batPath);
```

Comment:
```suggestion
					runtime.exec("cmd /c start \"\" \"" + batPath + "\"");
```

I think we need to quote the batch path.  Can you test this?